### PR TITLE
  Add flex container to add item view and css file

### LINKED ISSUE #57: 26. Style Add item view based on wire frame
AC:

* Make breathable space for elements
* Center add items `input`
* Center radio buttons and labels
* Style borders based on app color palette

Screen Shot 2021-02-24 at 8 51 02 PM

### PR DESCRIPTION
**Pull request by veeteeran**
## Description
This PR adds styles suing the Material UI framework to the form of the Add Item view

## Related Issue

closes #57 

## Acceptance Criteria

- [x] Make breathable space for elements
- [x] Center add items `input`
- [x] Center radio buttons and labels
- [ ] Style borders based on app color palette

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |

### Before
Screen Shot 2021-02-25 at 7 41 24 PM

### After
Screen Shot 2021-02-25 at 9 08 27 PM

## Testing Steps / QA Criteria
* One way to test the site is to click on the `details` button below to open the `Netlify preview`

Screen Shot 2021-01-12 at 6 05 51 PM

* Otherwise from your terminal, pull down this branch with `git pull origin VT-style-add-item-form` and check that branch out with `git checkout VT-style-add-item-form`
* Install the following dependencies for [React Font Awesome](https://fontawesome.com/how-to-use/on-the-web/using-with/react)
* npm i --save @fortawesome/fontawesome-svg-core
* npm install --save @fortawesome/free-solid-svg-icons
* npm install --save @fortawesome/react-fontawesome
* `npm start` to launch the app.
* Create a new list or join an existing list
* The list shown uses the token `stroll ames knack`

### CODE REVIEW

**Comment by veeteeran** [comment]
The form box should be centered on the page. I wanted to wait until @DeeDowns was finished with the header before doing that. Can I get suggestions to the overall `form` element on line 68. I'm trying to figure out if there is a better way using a Material UI component.

File: src/styles/AddItem.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  height: 25px;
+  padding-left: 5px;
+}
+
+label input:focus {
+  outline-color: #a1dffb;
+  outline-width: 3px;
+}
+
+/* fieldset {
```
Comment (MikeBLambert) [inline src/styles/AddItem.css:27]: Commented out code you may want to remove

File: src/styles/AddItem.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.container {
```
Comment (MikeBLambert) [inline src/styles/AddItem.css:1]: This CSS looks similar to what's in @DeeDowns PR and your PR's may clash with each other. You may want to touch base with each other to make sure that they play together well.

Comment (veeteeran) [reply src/styles/AddItem.css:1]: Oh, thanks!

File: src/components/AddItem.js [hunk 3]
```diff
@@ -2,6 +2,16 @@ import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { db } from '../lib/firebase';
 import Error from './Error';
+import TextField from '@material-ui/core/TextField';
```
Comment (MikeBLambert) [inline src/components/AddItem.js:5]: Nit: I'm guessing that in the @material-ui package, that these components are likely all exported from an index.js and you can probably clean up the imports a little by doing them like:

import { Radio, RadioGroup, FormControlLabel, ... } from "@material-ui/core";

File: src/components/AddItem.js [hunk 4]
```diff
@@ -54,60 +64,52 @@ const AddItem = ({ token }) => {
...
-        </button>
+          >
+            <FormControlLabel
+              value="7"
+              control={<Radio id="soon" />}
+              label="Soon"
+            />
+            <FormControlLabel
+              value="14"
+              control={<Radio id="kinda-soon" />}
```
Comment (MikeBLambert) [inline src/components/AddItem.js:95]: The radios look awesome and I love the slight animation that they come with :-)

**Review by MikeBLambert — APPROVED** [review]
The app is coming along and I love how the radio buttons look!

File: src/components/AddItem.js (hunk 4 above)
Comment (veeteeran) [reply src/components/AddItem.js:95]: Thanks Mike. I thought that list of imports was getting out of control!

File: src/components/AddItem.js (hunk 3 above)
Comment (veeteeran) [reply src/components/AddItem.js:5]: Thanks Mike. I thought that list of imports was getting out of control!

**Comment by veeteeran** [comment]
Also thanks @Lauren for the nudge to look into using Material UI components. That suggestion was in the back of mind my when I started styling and I think we can get a lot of it done leveraging that framework.

File: src/components/AddItem.js (hunk 3 above)
Comment (LINDGRENBA) [reply src/components/AddItem.js]: More imports !!!!!!!

**Review by LINDGRENBA — APPROVED** [review]
I think it looks awesome Viet! Nicely done.

File: src/components/AddItem.js [hunk 5]
```diff
@@ -54,62 +67,64 @@ const AddItem = ({ token }) => {
...
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      width={500}
+      height={300}
+      border={3}
+      borderColor="#d3d3d3"
+      borderRadius="borderRadius"
+    >
+      <form className="form-container" onSubmit={handleSubmit}>
```
Comment (veeteeran) [inline src/components/AddItem.js:80]: Can I get suggestions to the overall `form` element on line 80? I'm trying to figure out if there is a better way using a Material UI component.

**Pull request merged**